lib/matplotlib/figure.py

```diff
@@ -32,7 +32,7 @@
 
 from matplotlib.cbook import Stack, iterable
 
-from matplotlib import _image
+from matplotlib import image as mimage
```

Comment:
I think it's best to use our `mFOO` convention here.  I thought the import to a private name here was really confusing.  We could go through at some point and make `__all__` for the major modules with sane stuff, but I think that's a separate task that probably needs its own discussion about backward compatiblity/best practice etc.